utils.sh

```diff
@@ -49,6 +49,10 @@ function create_cluster() {
     mkdir -p ${assets_dir}/openshift
     cp -rf assets/generated/*.yaml ${assets_dir}/openshift
 
+    if [ ! -z "${ASSETS_EXTRA_FOLDER:-}" ]; then
+      cp -rf ${ASSETS_EXTRA_FOLDER}/* ${assets_dir}/openshift
```

Comment:
you want to make sure only yaml files are copied to the openshift directory or bootstrapping will fail (in a non obvious way)